framework/deproxy_client.py

```diff
@@ -419,6 +418,8 @@ def handle_read(self):
                         return
                     self.receive_response(response)
                     self.nrresp += 1
+                else:
+                    self.handle_read()
```

Comment:
It's temporary solution, therefore need to create new issue and add TODO comment with issue #id in it.